app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] According to the [requirements ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md)of this project, " `Ask for confirmation of email.` " , your application asks for confirmation of email. The `user.rb` file should be similar to: 
```
devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
```
Please kindly fix that.

Reply:
Thanks @Mihndim2020